part-iv-advanced-tuning/advanced-customizations.md

```diff
@@ -105,6 +105,8 @@ You can then sort these handlers based on their annotations by using the `annota
 For your HandlerEnhancer to run you'll need to create a `META-INF/services/org.axonframework.messaging.annotation.HandlerEnhancerDefinition` file containing
 the fully qualified class name of the handler enhancer you have created.
 
+> **Note** Since 3.3 Axon version it is possible to configure `HandlerDefinition` with Axon `Configuration`. If you are using Spring Boot defining `HandlerDefintion`s and `HandlerEnhancerDefinition`s as beans is sufficient (Axon autoconfiguration will pick them up and configure within Axon `Configuration`).
```

Comment:
I don't think we have to note that this is possible as off framework version 3.3, as users will only see this if they have RefGuide version 3.3 (or higher) opened up. It should go without saying.

Reply:
In future versions (3.4, 3.5, ...) you'd have to go through the past releases in order to find in which one this was added. We can remove it if you think it's not a valuable info.

Reply:
I think this reasoning would require us to retroactively add the version number to every feature which is added to the framework.

And why would you need to check in future versions when something was added?
The javadoc of a given class should provide the version number already, which I personally think is sufficient.